distributed/bokeh/worker_html.py

```diff
@@ -0,0 +1,34 @@
+import os
+
+from tornado import web
+
+dirname = os.path.dirname(__file__)
+
+
+class RequestHandler(web.RequestHandler):
+    def initialize(self, server=None, extra=None):
+        self.server = server
+        self.extra = extra or {}
+
+    def get_template_path(self):
+        return os.path.join(dirname, 'templates')
+
+
+class PrometheusHandler(RequestHandler):
+    def get(self):
+        import prometheus_client
+        #workers = prometheus_client.Gauge('memory_bytes',
```

Comment:
Is this comment to illustrate what could be added? If so it could be good to explicitly state that in the comment, however I'm not that keen on this approach.

Reply:
Will move and comment.